server/src/db/Seeder.js

```diff
@@ -1,14 +1,17 @@
 /* eslint-disable no-console */
 import { connection } from "../boot.js";
-import { Planet } from "../models/index.js"
+import { Planet, Review } from "../models/index.js"
 
 class Seeder {
   static async seed() {
     
     await Planet.query().insert({name: "Pluto", description: "No longer a planet!"})
     await Planet.query().insert({name: "Jupiter", description: "Very colorful planet."})
     await Planet.query().insert({name: "Saturn", description: "Planet with rings!"})
-    
+
+    const pluto = await Planet.query().findOne("name", "Pluto")
```

Comment:
for consistency, prefer calling the findOne method like this 
`.findOne({ name: "Pluto" })`